js/mexc3.js

```diff
@@ -3869,32 +3872,36 @@ module.exports = class mexc3 extends Exchange {
             throw new ExchangeError (this.id + ' toAccount must be one of ' + keys.join (', '));
         }
         const request = {
-            'currency': currency['id'],
+            'asset': currency['id'],
             'amount': amount,
-            'from': fromId,
-            'to': toId,
+            'fromAccountType': fromId,
+            'toAccountType': toId,
         };
-        const response = await this.spot2PrivatePostAssetInternalTransfer (this.extend (request, params));
+        if ((fromId === 'ISOLATED_MARGIN') || (toId === 'ISOLATED_MARGIN')) {
+            const symbol = this.safeString (params, 'symbol');
+            if (symbol === undefined) {
+                throw new ArgumentsRequired (this.id + ' transfer() requires a symbol argument for isolated margin');
+            }
+            const market = this.market (symbol);
```

Comment:
we're missing here omitting `symbol` from params, otherwise, it would override the unified value you're adding to `request`, and using unified symbols would not work. 

I will fix it, just a note for the future